test.sh

```diff
@@ -19,6 +19,32 @@ for dir in ${VERSIONS}; do
     VERSION=$dir test/run
   fi
 
+  if [ -n "${TEST_CONU_MODE}" ]; then
+    if [[ -x test/run-conu ]]; then
+      if [ -n "$CONU_IMAGE" ]; then
+        docker run \
+          --net=host \
+          -e VERSION=$dir \
+          -e IMAGE_NAME \
+          --rm -v /dev:/dev:ro \
+          -v /var/lib/docker:/var/lib/docker:ro \
+          --security-opt label=disable \
+          --cap-add SYS_ADMIN \
+          -ti \
+          -v /var/run/docker.sock:/var/run/docker.sock \
+          -v ${PWD}/../:/src \
+          -w /src/${dir}/ \
+          -ti \
+          $CONU_IMAGE \
+          ./test/run-conu
+      else
+        VERSION=$dir ./test/run-conu
```

Comment:
One pedantic note, can you please double-quote the shell vars above?

Reply:
Yes, I can.